src/vconsole/vconsole-setup.c

```diff
@@ -516,19 +516,27 @@ static void setup_remaining_vcs(int src_fd, unsigned src_idx, bool utf8) {
 }
 
 static int find_source_vc(char **ret_path, unsigned *ret_idx) {
-        int r, err = 0;
+        int err = 0;
 
         assert(ret_path);
         assert(ret_idx);
 
+        /* This function returns an fd when it finds a candidate. When it fails, it returns the first error
+         * that occured when the VC was being opened or -EBUSY when it finds some VCs but all are busy
+         * otherwise -ENOENT when there is no allocated VC. */
+
         for (unsigned i = 1; i <= 63; i++) {
                 _cleanup_close_ int fd = -EBADF;
                 _cleanup_free_ char *path = NULL;
+                int r;
```

Comment:
as per coding style: `int r;` is special to us. It's the primary variable we almost always write error returns to. Hence it gets this special privilege to always be defined at outermost scope. Even though our usual rule is to define variables always at innermost scope. `int r;` is exempted from that rule. Please move this back up